project_euler/problem_13/sol2.py

```diff
@@ -0,0 +1,5 @@
+sum = 0
+with open("num.txt",'r') as f:
+    for line in f:
+        sum += int(line)
```

Comment:
Please consider reimplementing as a list comprehension that uses the Python builtin [__sum()__](https://docs.python.org/3/library/functions.html#sum).

Reply:
Python builtin [__divmod__](https://docs.python.org/3/library/functions.html#divmod)

Reply:
Python builtin [__divmod__](https://docs.python.org/3/library/functions.html#divmod)